README.md

```diff
@@ -1,38 +1,45 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Title-Info
-> This project is about a ........
+# Second Capstone JavaScript
+> This is a web app where users can view all pokemons with their characteristics, abilities and types. Data is fetched from the Pokeapi. Here user can like, comment and view like and comments of him/her and others.
 
 
-# ScreenShots/GIF
-
+# GIF
+- ![](cap2.gif)
 
 ## Built With
 - HTML
 - Javascript
+- CSS
+- Webpack
 - VScode
+- Jest
 
 ## Live Demo
-For the link to the live demo, [Click here]()
+For the link to the live demo, [Click here](https://tufahel.github.io/second-capstone/)
 
 ## Getting Started
 To get a local copy up and running you just need to follow the following steps;
 - Clone this repository with
-git clone https://github.com/Tufahel using your terminal or command line.
+git clone https://github.com/Tufahel/second-capstone.git using your terminal or command line.
 - run code . to open it in vscode
+- run `npm install` in terminal
+- run `npm start`
+Now you are ready to use this web app.
```

Comment:
- [ ] In the Readme file Please add the command to test the project as well. [ Hint: `npm test`]

Reply:
Thank you for your review, we have updated it.